bin/collator/src/cli.rs

```diff
@@ -31,6 +31,7 @@ pub struct Cli {
     pub run: cumulus_client_cli::RunCmd,
 
     /// Enable EVM tracing module on a non-authority node.
+    #[cfg(feature = "evm-tracing")]
```

Comment:
Instead of adding options one by one, and having `#[cfg(feature = "evm-tracing")]]` over each, you should create a struct and flatten it. Perfect example is few lines above for the `cumulus_client_cli`.

Reply:
Preprocessor conditions makes it possible to exclude tracing dependencies from production client.

Reply:
That is correct. However, I didn't comment that at all.

My comment relates to having **many** feature checks and I suggest to add only one instead and use the flatten macro.